user/getting-started.md

```diff
@@ -4,38 +4,107 @@ layout: en
 permalink: /user/getting-started/
 ---
 
-The very short guide to using Travis CI with your GitHub hosted code repository. Totally new to all this? Try reading [Travis CI for Complete Beginners](/user/for-beginners) instead.
+The very short guide to using Travis CI with your GitHub hosted code repository. If you're new to continuous integration or would like some more information on what Travis CI does, start with [Travis CI for Complete Beginners](/user/for-beginners) instead.
 
-Or if you want a more complete guide to a particular language, pick one of these:
+<div id="toc"></div>
 
-{% include languages.html %}
+## To get started with Travis CI
 
-### To get started with Travis CI:
+You'll need a GitHub account with admin access to at least one repository, and ideally you'll already have a working build script that you run manually.
 
 1. [Sign in to Travis CI](https://travis-ci.org/auth) with your GitHub account, accepting the GitHub [access permissions confirmation](/user/github-oauth-scopes).
 
-2. Once you're signed in, and we've synchronized your repositories from GitHub, go to your [profile page](https://travis-ci.org/profile) and enable Travis CI for the repository you want to build.
+2. Once you're signed in, and we've synchronized your GitHub repositories, go to your [profile page](https://travis-ci.org/profile) and enable Travis CI for the *open source* repository you want to build. If you want to build a private repository, sign in to [Travis CI for private repositories](https://travis-ci.com/profile) instead.
```

Comment:
I think the comment about admin access being required should show up here? It's a good thing to note in my opinion - not sure we need to get rid of it?